definitions/environments/node/flow_v0.261.x-/node.js

```diff
@@ -1307,6 +1313,70 @@ declare module "fs" {
   declare function copyFile(src: string, dest: string, flags?: number, callback: (err: ErrnoError) => void): void;
   declare function copyFileSync(src: string, dest: string, flags?: number): void;
 
+  declare type GlobOptions<WithFileTypes: boolean> = $ReadOnly<{
+    /**
+     * Current working directory.
+     * @default process.cwd()
+     */
+    cwd?: ?string | void,
+    /**
+     * `true` if the glob should return paths as `Dirent`s, `false` otherwise.
+     * @default false
+     * @since v22.2.0
+     */
+    WithFileTypes?: WithTypes,
```

Comment:
```suggestion
    WithFileTypes?: WithFileTypes,
```
Looks like `WithTypes` is not referenced.